lib/marked-renderer.js

```diff
@@ -0,0 +1,13 @@
+// A custom renderer that avoids wrapping <img> tags in <p> tags
```

Comment:
Not a blocker in the slightest, but this would look nice as JSDoc above the class?

Makes the comment show up in other files when you mouse hover 🙌

Reply:
Fancy!